src/core/utils.ts

```diff
@@ -191,3 +191,35 @@ export function onDOMReady(fn: () => void) {
         document.addEventListener('DOMContentLoaded', fn);
```

Comment:
Can't comment on the actual line :D but will isBrowserSupported be modified to take enabled features?

Reply:
It could be, right now the check is done as `if (!isBrowserSupported() && !configHasDdlFeature(init[1])) {` but will roll that up inside the actual `isBrowserSupported` check